src/hist/plot.py

```diff
@@ -6,12 +6,18 @@
 
 import hist
 
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
```

Comment:
Generally I like to bundle these conditional imports into a `typing.py` file in the package, but I can do that later.

Reply:
Oh, I take that back, there is one already. Please move this to src/hist/typing.py and import from there.